nemo/collections/nlp/metrics/qa_metrics.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+# Copyright 2019 The Google Research Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import string
+from collections import Counter
+
+
+class QAMetrics(object):
+    @classmethod
+    def normalize_answer(cls, s: str):
+        """ Lower text and remove punctuation, articles and extra whitespace """
+
+        def remove_articles(text):
```

Comment:
unless there are good reasons (i.e. needing to access local variables), I wouldn't recommend putting methods in a method. This is because they make them harder to test and also if you want to re-use the method somewhere else later on, this is pretty much impossible